pkg/3scale/amp/operator/zync_options_provider.go

```diff
@@ -266,6 +278,12 @@ func (z *ZyncOptionsProvider) zyncDatabasePodTemplateLabels() map[string]string
 		labels[k] = v
 	}
 
+	if z.apimanager.Spec.Zync.DatabaseLabels != nil {
```

Comment:
this `if` is not needed

Reply:
Addressed in this [commit](https://github.com/3scale/3scale-operator/pull/835/commits/f903bf41523372ad2297594e4efac295a12c0665)